analysis/src/topObjectSelection.cc

```diff
@@ -33,14 +32,12 @@ vector<TParticle> topObjectSelection::elecSelection() {
 vector<TParticle> topObjectSelection::muonSelection() {
   vector<TParticle> muons; 
   for (UInt_t i = 0; i < nMuon; ++i){
-    if (!Muon_tightId[i]) continue;
-    if (Muon_pt[i] < 20) continue;
-    if (isSemiLep) { if (Muon_pt[i] < 26) continue; }
-    if (std::abs(Muon_eta[i]) > 2.4) continue;
-    if (isSemiLep) { if (std::abs(Muon_eta[i]) > 2.1) continue; }
-    if (Muon_pfRelIso04_all[i] > 0.15) continue;
-    if (!Muon_globalMu[i]) continue;
```

Comment:
Do we need these lines or is it covered by ID?

Reply:
(Ah, sorry for the wrong touch...)
It's covered the following replacements, except for the last line (`Muon_globalMu`). I thought that it's already in the tight ID cut, so I didn't consider

Reply:
Okay, does anyone have more comments, or shall I just pull this in?

Reply:
I have no additional comment